libd-llvm/src/d/llvm/type.d

```diff
@@ -310,7 +310,32 @@ final class TypeGen {
 
 		import std.string;
 		auto llvmStruct = typeSymbols[i] = LLVMStructCreateNamed(llvmCtx, i.mangle.toStringz());
+
+		LLVMValueRef[] vtbl = [];
+		foreach (member; i.members) {
+			if (auto m = cast(Method) member) {
+				vtbl ~= pass.visit(m);
```

Comment:
That does not make any sense, why are you generating the vtable value here ? You can only generate vtable on the classes that implement the interface.